library/model/author.py

```diff
@@ -1,23 +1,24 @@
 class Author:
 
-    firstname: str
-    lastname: str
+    _firstname: str
+    _lastname: str
 
```

Comment:
To make the code symmetric, you could additionally declare _uid as a member variable here.